src/building.py

```diff
@@ -1,23 +1,46 @@
+# pylint: disable=missing-module-docstring
+# pylint: disable=unnecessary-pass
```

Comment:
Why are we disabling this?

Reply:
i think because now every file is structured as a module pylint requires docstrings at line 1 of every file, not really necessary for us i thought so I just disabled it. And the unnecessary pass error kept appearing from the abstract base classes, it doesnt affect anything just pylint being annoying